src/jit/morph.cpp

```diff
@@ -16851,6 +16857,8 @@ void Compiler::fgMorph()
     /* For x64 and ARM64 we need to mark irregular parameters early so that they don't get promoted */
     fgMarkImplicitByRefArgs();
 
+    EndPhase(PHASE_MORPH4);
```

Comment:
I don't know how short we want to keep these names, but `PHASE_MORPH_IMPLICIT_BYREF` might be more meaningful than just `PHASE_MORPH_BYREF`